lib/bookmark_query.rb

```diff
@@ -36,7 +36,9 @@ def list_all
     results = results.merge(Post.secured(@guardian))
 
     if @params[:q].present?
-      results = results.where("bookmarks.name ILIKE :q OR posts.raw ILIKE :q", q: "%#{@params[:q]}%")
+      results = results
+        .joins("LEFT JOIN post_search_data ON post_search_data.post_id = bookmarks.post_id")
+        .where("bookmarks.name ILIKE :q OR post_search_data.raw_data ILIKE :q", q: "%#{@params[:q]}%")
```

Comment:
`post_search_data.raw_data ILIKE :q` isn't right. Do have a read through https://www.postgresql.org/docs/12/textsearch.html and https://github.com/discourse/discourse/blob/master/lib/search.rb#L790-L802 to have an idea of how text search in PG works.